python/python/lancedb/query.py

```diff
@@ -1727,10 +1744,22 @@ class AsyncFTSQuery(AsyncQueryBase):
     def __init__(self, inner: LanceFTSQuery):
         super().__init__(inner)
         self._inner = inner
+        self._reranker = RRFReranker()
 
     def get_query(self):
         self._inner.get_query()
 
+    def rerank(
+        self,
+        reranker: Reranker = RRFReranker(),
+    ) -> AsyncFTSQuery:
+        if reranker and not isinstance(reranker, Reranker):
+            raise ValueError("reranker must be an instance of Reranker class.")
+
+        self._reranker = reranker
+
+        return AsyncFTSQuery(self._inner.rerank(reranker))
```

Comment:
calling `self._inner.rerank` calls into the underlying rust implementation of `FTSQuery`:
https://github.com/lancedb/lancedb/blob/50c30c5d34dab481ca00897ed9fb63aa8b033567/python/src/query.rs#L146

But since there's no `rerank` method on this struct, I don't think this call will succeed. 

Probably instead we'd want to do something like:

```suggestion
        self._reranker = reranker
        return self
```

Reply:
This is also why the unit test is failing

Reply:
thanks! now it all makes sense :)